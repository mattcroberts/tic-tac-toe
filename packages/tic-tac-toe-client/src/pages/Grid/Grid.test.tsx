import * as React from "react";
import { shallow } from "enzyme";
import GridPage from ".";

describe("GridPage", () => {
    const defaultProps: any = {};
    it("should render", () => {
        const component = shallow(<GridPage {...defaultProps} />);

        expect(component).toMatchSnapshot();
    });
});
