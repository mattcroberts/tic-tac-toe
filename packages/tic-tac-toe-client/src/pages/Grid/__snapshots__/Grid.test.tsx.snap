// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridPage should render 1`] = `
<Component>
  <Grid
    isDraw={false}
    onItemClick={[Function]}
    winner={null}
  />
</Component>
`;
